authmodule/src/main/webapp/index.jsp

```diff
@@ -0,0 +1,5 @@
+<html>
```

Comment:
indentation of xml

Reply:
All HTML pages now follow VS-Code standards.